pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -341,14 +341,6 @@ buildStdenv.mkDerivation ({
     gappsWrapperArgs+=(--argv0 "$out/bin/.${binaryName}-wrapped")
   '';
 
-  postFixup = lib.optionalString buildStdenv.isLinux ''
```

Comment:
Does this break executing firefox without the wrapper? If so this sound wrong. I've never really used the wrapper much...

Reply:
No, it still executes and functions as a web browser. Both are optional runtime libraries and firefox can function fine without them. That being said it does degrade the feature set provided by firefox-unwrapped. Namely firefox goes back to using non-native notifications and there is no more screensaver inhibitor. I am not sure if that is a problem though since firefox-unwrapped already lacks things like VA-API, audio, etc.